gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-agp = "7.3.1"
-androidx-activity = "1.6.0"
-androidx-appcompat = "1.6.0-rc01"
+agp = "7.4.0"
+androidx-activity = "1.6.1"
+androidx-appcompat = "1.6.0"
 androidx-browser = "1.4.0"
 androidx-cardview = "1.0.0"
 androidx-constraintlayout = "2.1.4"
 androidx-core = "1.9.0"
 androidx-exifinterface = "1.3.4"
-androidx-fragment = "1.5.3"
+androidx-fragment = "1.5.5"
```

Comment:
AndroidX dependencies are already being upgraded in https://github.com/tuskyapp/Tusky/pull/3169

Reply:
f84c5b980e78da12c4828dbad8aa4e59ed37f02c